test/new-e2e/pkg/utils/infra/stack_manager.go

```diff
@@ -121,13 +147,12 @@ func (sm *StackManager) DeleteStack(ctx context.Context, name string) error {
 // ForceRemoveStackConfiguration removes the configuration files pulumi creates for managing a stack.
 // It DOES NOT perform any cleanup of the resources created by the stack. Call `DeleteStack` for correct cleanup.
 func (sm *StackManager) ForceRemoveStackConfiguration(ctx context.Context, name string) error {
-	sm.lock.Lock()
-	defer sm.lock.Unlock()
 
-	stack, ok := sm.stacks[name]
+	storedStack, ok := sm.stacks.Get(name)
 	if !ok {
 		return fmt.Errorf("unable to remove stack %s: stack not present", name)
 	}
+	stack := storedStack
```

Comment:
```suggestion
	stack, ok := sm.stacks.Get(name)
	if !ok {
		return fmt.Errorf("unable to remove stack %s: stack not present", name)
	}
```
Should be enough

Reply:
Yes that's a leftover from previous tests I made, I though I had delete everything